content/messaging/customer-chat-api/changelog/index.mdx

```diff
@@ -146,3 +147,4 @@ This document is the record of all the changes in the **Customer Chat API**, Web
 ### Removed
 
 - The **Send File** method was removed.
+- The [`customer_updated`](/management/configuration-api/v3.1/#customer_updated) push was removed from **Set Customer Fields** method
```

Comment:
I wouldn't classify it as removed because this push wasn't completely removed. It's no longer triggered by Set Customer Fields, so I'd approach it as a change.

### Changed

- (...)
- The [**customer_updated**](/messaging/customer-chat-api/v3.1/rtm-reference/#customer_updated) push is no longer triggered by the **Set Customer Fields** method.